src/coreclr/src/debug/ee/controller.cpp

```diff
@@ -5644,9 +5645,56 @@ bool DebuggerStepper::TrapStepInHelper(
     return false;
 }
 
-FORCEINLINE bool IsTailCall(const BYTE * pTargetIP)
+static bool IsTailCallJitHelper(const BYTE * ip)
```

Comment:
static seems strange here. What was the reasoning for adding it?

Reply:
Also why did we remove forceinline?

Reply:
I added another function (below) which should definitely not be `FORCEINLINE`, so I just changed both of them to have the consistent signatures I would expect, i.e. leave the inlining decisions up to the C++ compiler.

They are static because they are only used as helper functions from within this file so they can have internal linkage. The compiler will be able to remove `IsTailCall` when it's inlined and does not have more usages due to this, and the other function I added will not be visible to the linker.